### PR TITLE
Add missing includes

### PR DESCRIPTION
**Pull request by Flamefire**
Fix some missing include and remove outdated ones
Also adds namespace to LogLvl

### CODE REVIEW

File: src/libPMacc/include/debug/VerboseLogMakros.hpp [hunk 1]
```diff
@@ -32,7 +30,7 @@
  * @param name name of the log lvl, name is needet later to call log<name>(...)
  */
 #define DEFINE_LOGLVL(code,name)                            \
-    typedef LogLvl < code, thisClass > name;                \
+    typedef PMacc::LogLvl < code, thisClass > name;                \
```
Comment (ax3l) [inline src/libPMacc/include/debug/VerboseLogMakros.hpp]: minor: `\` alignment :)

**Comment by ax3l** [comment]
should affect the master (-> change log entry) but does not need a backport

**Comment by Flamefire** [comment]
Fixed alignment as suggested by @ax3l

**Comment by ax3l** [comment]
thx!

**Comment by ax3l** [comment]
@Flamefire can you pls add your compiler versions in the PR description? since you probably included some files from libPMacc and got a compile error, I would list that in the change log.

**Comment by Flamefire** [comment]
@ax3l Done. But This should be independent of the compiler and shown when you include the files in a different order then done in picongpu.

**Comment by ax3l** [comment]
all right, thanks for the update. so basically it's just a plain "missing include" when these files were used (that caused a compile error).

**Comment by Flamefire** [comment]
Correct

**Pull request merged**